src/DisplayHelper/DisplayData.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace TheRealMVP\DisplayHelper;
```

Comment:
This is outrageously picky, but could you change DisplayHelper to be plural to match your other folders/namespaces?